### PR TITLE
#2 - Update Readme

### LINKED ISSUE #2: Adjust links in README.md
## Acceptance criteria

- [ ] The README.md file is updated with links as specified in the instructions.

## Details

See the README.md file in the repo for details.

### PR DESCRIPTION
**Pull request by BK1031**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,19 @@
-# STARTER-team01
+# w24-4pm-2-team01
 
 Lab instructions: <https://ucsb-cs156.github.io/w24/lab/team01.html>
 
-* TODO: Update the following table, then delete this line that says "TODO".
+| Name         | GitHub Id                                           |  Service                    | Controller                |
+|--------------|-----------------------------------------------------|-----------------------------|---------------------------|
+| Bharat Kathi | [bk1031](https://github.com/bk1031)                 | `JokeQueryService`      | `JokeController`      |
```
Comment (kylebkumar) [inline README.md]: Please align this line (if that's what the instructions say)

**Review by kylebkumar — CHANGES_REQUESTED** [review]
Please address comment

File: README.md (hunk 1 above)
Comment (BK1031) [reply README.md]: Reformatted the table, markdown renders correctly either way 👍

**Review by kylebkumar — APPROVED** [review]
LGTM

**Pull request merged**